keyboards/dz60/keymaps/LEdiodes/LEdiodes.h

```diff
@@ -0,0 +1,19 @@
+#ifndef dz60_H
+#define dz60_H
+#include "quantum.h"
+
+#define KEYMAP( \
```

Comment:
What is different with this LAYOUT as compared to the other LAYOUTS in existence in https://github.com/qmk/qmk_firmware/blob/master/keyboards/dz60/dz60.h

Reply:
nothing, I was just dinking around I guess. I have updated the code to use the default dz60.h.